addons/web_editor/static/src/js/editor/odoo-editor/src/OdooEditor.js

```diff
@@ -4155,7 +4155,7 @@ export class OdooEditor extends EventTarget {
             node.replaceChildren();
         }
 
-        sanitize(element, this.editable);
+        sanitize(element, editable || this.editable);
```

Comment:
So with the above change this can become
```suggestion
        sanitize(element, editable);
````